### PR TITLE
Added a small banner for zap in edu

### PR DESCRIPTION
**Pull request by lisaqyli**
Screen Shot 2022-02-12 at 8 47 06 AM

### CODE REVIEW

**Review by kathaypacific — APPROVED** [review]
Nice! I think the card colour looks a bit different from the new colours but i'm okay if you like how it looks like this :)

File: public/locales/en.json [hunk 1]
```diff
@@ -225,5 +225,7 @@
   "zap": "Zap in",
   "zapInAmount": "Zap in amount",
   "fees": "Fees",
-  "feesInfo": "Some explanation about where the fees come from"
+  "feesInfo": "Some explanation about where the fees come from",
+  "ZapInEdu": "What is Zap in?",
+  "ZapInEduDesc": "Zap in is a simplication of the normally complicated yield farming process. You can simply swap any token to a auto-compounding yield farming token in one click."
```
Comment (cajubelt) [inline public/locales/en.json]:
```suggestion
  "ZapInEduDesc": "\"Zap in\" allows you to swap any token (like cUSD or CELO) for a share in a yield farm with automatic compound interest. Learn more about DeFi on Revo [here](https://docs.revo.market/)"
```

Comment (cajubelt) [reply public/locales/en.json]: I felt like "simplication of the normally complicated yield farming process" says more about the status quo than about us. If people know about the status quo, they don't need this, and if they don't, it won't mean as much to them.

**Comment by cajubelt** [comment]
I updated the copy and styling, plus a link to our docs (this has the side benefit of letting us say we comply with bakeoff requirements)

Screen Shot 2022-02-15 at 6 26 47 PM

**Pull request merged**